localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,344 @@
+
+ ### Wildhammer ###
+ # Feudal
+ chief_male_wildhammer:0 "Chief"
+ chieftess_female_wildhammer:0 "Chieftess"
+ chiefdom_wildhammer:0 "Chiefdom"
+ thane_male_wildhammer:0 "Thane"
+ thane_female_wildhammer:0 "Thane"
+ high_chiefdom_wildhammer:0 "High Chiefdom"
+ high_thane_male_wildhammer:0 "High Thane"
+ high_thane_female_wildhammer:0 "High Thane"
+
+ ### Dark Iron ###
+ # Feudal
+ lord_male_darkiron:0 "Lord"
+ lady_female_darkiron:1 "Lady"
+ lordship_darkiron:0 "Lordship"
+ thane_male_darkiron:0 "Thane"
+ thane_female_darkiron:0 "Thane"
+
+ ### Old Dwarven / Frostborn ###
+ # Feudal
+ chief_male_old_dwarven:0 "Chief"
+ chieftess_female_old_dwarven:0 "Chieftess"
+ chiefdom_old_dwarven:0 "Chiefdom"
+ thane_male_old_dwarven:0 "Thane"
+ thane_female_old_dwarven:0 "Thane"
+ high_chiefdom_old_dwarven:0 "High Chiefdom"
+ high_thane_male_old_dwarven:0 "High Thane"
+ high_thane_female_old_dwarven:0 "High Thane"
+ high_king_male_old_dwarven:0 "High King"
+ high_queen_female_old_dwarven:0 "High Queen"
+ high_kingdom_old_dwarven:0 "High Kingdom"
+
+ ### Vrykul and Kvaldir ###
+ # Feudal
+ thane_male_vrykul:0 "Thane"
+ thane_female_vrykul:0 "Thane"
+ chiefdom_vrykul:0 "Chiefdom"
+ jarl_male_vrykul:0 "Jarl"
+ frue_female_vrykul:0 "Frue"
+ jarldom_vrykul:0 "Jarldom"
+ high_king_male_vrykul:0 "High King"
+ high_queen_female_vrykul:0 "High Queen"
+ high_kingdom_vrykul:0 "High Kingdom"
+
+ ### Frost Vrykul ###*
+ # Feudal
+ thane_male_frost_vrykul:0 "Thane"
+ thane_female_frost_vrykul:0 "Thane"
+ chiefdom_frost_vrykul:0 "Chiefdom"
+ overthane_male_frost_vrykul:0 "Overthane"
+ overthane_female_frost_vrykul:0 "Overthane"
+ jarldom_frost_vrykul:0 "Jarldom"
+ high_king_male_frost_vrykul:0 "High King"
+ high_queen_female_frost_vrykul:0 "High Queen"
+ high_kingdom_frost_vrykul:0 "High Kingdom"
+
+ ### Staroveric ###
+ # Feudal
+ grand_prince_male_solstice:0 "Grand Prince"
+ grand_princess_female_solstice:0 "Grand Princess"
+ grand_principality_solstice:0 "Grand Principality"
+ tsar_male_solstice:0 "Tsar"
+ tsaritsa_female_solstice:0 "Tsaritsa"
+ tsardom_solstice:0 "Tsardom"
+
+ ### Dryad group ###
+ # Feudal / Republic / Clan / Theocratic / Tribal
+ tender_male_dryad_group:0 "Tender"
+ tender_female_dryad_group:0 "Tender"
+ refuge_dryad_group:0 "Refuge"
+ guardian_male_dryad_group:0 "Guardian"
+ guardian_female_dryad_group:0 "Guardian"
+ grove_dryad_group:0 "Grove"
+ lord_male_dryad_group:0 "Lord"
+ lady_female_dryad_group:0 "Lady"
+ grand_grove_dryad_group:0 "Grand Grove"
+ high_lord_male_dryad_group:0 "High Lord"
+ high_lady_female_dryad_group:0 "High Lady"
+ domain_dryad_group:0 "Domain"
+ king_male_dryad_group:0 "King"
+ queen_female_dryad_group:0 "Queen"
+ kingdom_dryad_group:0 "Kingdom"
+
+ ### Tol'vir ###
+ # Feudal - Normal
+ governor_male_tolvir:0 "Governor"
+ governor_female_tolvir:0 "Governor"
+ region_tolvir:0 "Region"
+ nomarch_male_tolvir:0 "Nomarch"
+ nomarch_female_tolvir:0 "Nomarch"
+ nomes_tolvir:0 "Nomes"
+ pharaoh_male_tolvir:0 "Pharaoh"
```

Comment:
Tol'vir is led by king (https://wowpedia.fandom.com/wiki/King_Phaoris).

Reply:
Yes, but Pharaoh does exist, and since Tekahn becomes "Dark Pharaoh" when under Deathwing's orders I think it fits better that it's Pharaoh rather than Kings. In addition, it adds diversity once again.
https://wowpedia.fandom.com/wiki/Pharaoh_Tebhotep